.github/workflows/test.yml

```diff
@@ -16,6 +17,11 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
-      # You can test your matrix by printing the current Python version
-      - name: Display Python version
-        run: python -c "import sys; print(sys.version)"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          # pip install -r requirements.txt
```

Comment:
Does the # here mean comment like it does in Python, or is that something else?  If it is comment, why is this line commented out?

Reply:
Oh wait, is it because there's not a requirements.txt file yet? 😄

Reply:
Correct, # is for comments. There is no requirements.txt yet because nothing we are importing needs to be installed by pip, all of those are native to python.